.github/workflows/assign-issue-to-project.yaml

```diff
@@ -1,34 +0,0 @@
-# https://github.com/marketplace/actions/assign-to-one-project#:~:text=GitHub%20Action%20for%20Assign%20to,columns%20in%20your%20project%20dashboard.
-name: Assign Issue to Airbyte Github Project
-
-on:
-  issues:
-    types: [opened]
-env:
-  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-jobs:
-  assign_one_project:
-    runs-on: ubuntu-24.04
-    name: Assign to Airbyte Github Project
-    steps:
-      - name: Assign documentation issues to the Documentation Roadmap project
-        uses: srggrs/assign-one-project-github-action@1.2.0
-        if: |
-          contains(github.event.issue.labels.*.name, 'area/documentation') ||
-          contains(github.event.pull_request.labels.*.name, 'area/documentation')
-        with:
-          project: "https://github.com/airbytehq/airbyte/projects/4"
-          column_name: "Needs Triage"
-      - name: Assign New Community Issues to Github Project
-        uses: srggrs/assign-one-project-github-action@1.2.0
-        if: contains(github.event.issue.labels.*.name, 'community')
-        with:
-          project: "https://github.com/airbytehq/airbyte/projects/16"
-          column_name: "To do"
-      - name: Assign New Community PR to Github Project
```

Comment:
Are we sure @marcosmarxm is not using this project to manage contributions?

Reply:
You can remove it.

Reply:
We are sure, because we have checked that the project with ID 4 is ancient history.